docs/documentation/server_admin/topics/organizations/authenticating-members.adoc

```diff
@@ -106,6 +106,24 @@ Change the *first broker login* flow by following these steps:
 .Organizations first broker flow
 image:images/organizations-first-broker-flow.png[alt="Organizations first broker flow"]
 
-
 You should now be able to authenticate using any identity provider associated with an organization
 and have the user joining the organization as a member as soon as they complete the first browser login flow.
+
+== Configuring how users authenticate
+
+If the flow supports organizations, you can configure some of the steps to change how users authenticate to the realm.
+
+You are able to configure the following behaviors:
+
+* Enforce that users are members of an organization
+
+=== Enforcing that users are members of an organization
+
+Some use cases will require users to authenticate to a realm only if they are a member of any or a specific organization in the realm.
+
```

Comment:
```suggestion
For example, some use cases will require users to authenticate to a realm only if they are a member of any or a specific organization in the realm.

```

Reply:
I am not sure you need the text I have suggested you delete. Let me know if this makes sense.

Reply:
Thanks. Updated.